apps/judicial-system/backend/src/app/modules/case/test/caseController/getRulingPdf.spec.ts

```diff
@@ -52,15 +63,20 @@ describe('CaseController - Get ruling pdf', () => {
     const theCase = { id: caseId } as Case
     const res = {} as Response
 
-    beforeEach(async () => {
+    it('should lookup pdf', async () => {
```

Comment:
Please preserve the test pattern used and only call givenWhenThen in a beforeEach clause.